CHANGELOG.md

```diff
@@ -32,6 +32,22 @@
     end
     ```
 
+- Support `Sentry::Transaction#set_measurement` [#1838](https://github.com/getsentry/sentry-ruby/pull/1838)
+
+    Usage:
+
+    ```rb
+    # enable this experimental feature first
+    Sentry.init do |config|
+      # other configs
+      config.experiments.custom_measurements = true
```

Comment:
this doesn't need to be experimental anymore tbh, do you want to remove this logic or keep it for later experimental features?